packages/common/src/types/collection.ts

```diff
@@ -76,7 +83,9 @@ export type TCollectionRequest = TDatabaseRequest &
  */
 export type TCollectionCreateRequest = TCollectionRequest &
   Pick<TCollection, 'schema'> &
-  Pick<OwnershipAndPermission, 'permission' | 'group'>;
+  Pick<OwnershipAndPermission, 'permission' | 'group'> & {
+    collectionIndex?: TCollectionIndex[];
```

Comment:
Make sure there is no derived type from `TCollectionCreateRequest`, since based type shouldn't contain optional fields.

Reply:
Yeah, I've checked, `TCollectionCreateRequest` is a special type compare to another `TCollection` type because it need `index`

Reply:
I resolved, using TPickAlter to rename group to groupName